projects/pre-sentence-reports-to-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/integrations/delius/presentencereport/PreSentenceReportService.kt

```diff
@@ -0,0 +1,18 @@
+package uk.gov.justice.digital.hmpps.integrations.delius.presentencereport
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.exception.NotFoundException
+import uk.gov.justice.digital.hmpps.integrations.delius.courtreport.CourtReportRepository
+
+@Service
+class PreSentenceReportService(
+    private val courtReportRepository: CourtReportRepository,
+    private val objectMapper: ObjectMapper
+) {
+    fun getPreSentenceReportContext(reportId: String): PreSentenceReportContext {
+        val json = courtReportRepository.getCourtReportContextJson(reportId)
+            ?: throw NotFoundException("CourtReport", "id", reportId)
+        return objectMapper.readValue(json, PreSentenceReportContext::class.java)
```

Comment:
Are we definitely happy with this approach?  i.e. creating JSON in the database, deserializing into a PreSentenceReportContext object, then Spring re-serializing it to send in the response?